pkg/network/protocols/http/etw_http_service.go

```diff
@@ -742,6 +744,8 @@ func httpCallbackOnHTTPRequestTraceTaskParse(eventInfo *etw.DDEventRecord) {
 		if len(urlParsed.Path) == 0 {
 			urlParsed.Path = "/"
 		}
+		httpConnLink.urlPath = urlParsed.Path
```

Comment:
the RequestFragment is constructed by us so that it matches what the shared code is expecting later in the processing.
it matches what the linux/ebpf side uses.

Reply:
right